consensus/src/round_manager.rs

```diff
@@ -91,6 +98,23 @@ impl UnverifiedEvent {
                 cd.verify(validator)?;
                 VerifiedEvent::CommitDecision(cd)
             }
+            UnverifiedEvent::FragmentMsg(f) => {
+                f.verify(peer_id, quorum_store_enabled)?;
```

Comment:
Will it be cleaner to check quorum_store_enabled before? We know already here that if it is false we should not even got quorum store messages.
Maybe check it in process_message in epoch_manager?